feat: Add USE_ROUNDED_CORNERS configuration option

Kingcatz opened the pull request:
- Add USE_ROUNDED_CORNERS setting to control video element styling
- Create context processor to pass UI settings to templates
- Implement inline CSS approach for rounded corners on video elements
- Add responsive design (no rounded corners on mobile)
- Default: enabled (6px border radius on desktop)
- Can be disabled by setting USE_ROUNDED_CORNERS = False in local_settings.py

Uses inline CSS method following MediaCMS implementation pattern.

## Description
This replicates a feature from MediaCMS that was not integrated into CinemataCMS. Implementing it would be a good way to give a slightly new look to Cinemata. 

## Related Issue
<!--- Please link to the issue here: -->
Fixes #110 110

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->

## How Has This Been Tested?
Tested on Ubuntu 22.0.4 using Chrome, Brave and Firefox. 

## Screenshots (if appropriate):

<img width="2553" height="1486" alt="Screenshot from 2025-07-10 17-29-56" src="https://github.com/user-attachments/assets/2fb70fde-387f-4a8a-a864-906b1f586792" />
<img width="2553" height="1486" alt="Screenshot from 2025-07-10 17-29-45" src="https://github.com/user-attachments/assets/a49803ef-28ee-4541-9b55-8604edc16838" />


## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Kingcatz committed:
feat: Add USE_ROUNDED_CORNERS configuration option

- Add USE_ROUNDED_CORNERS setting to control video element styling
- Create context processor to pass UI settings to templates
- Implement inline CSS approach for rounded corners on video elements
- Add responsive design (no rounded corners on mobile)
- Default: enabled (6px border radius on desktop)
- Can be disabled by setting USE_ROUNDED_CORNERS = False in local_settings.py

Uses inline CSS method following MediaCMS implementation pattern.

jmcruz14 reviewed (commented):
<img width="415" height="529" alt="Screenshot 2025-07-11 at 3 29 19 AM" src="https://github.com/user-attachments/assets/0a360f07-d684-4a9f-bf79-18a972c2fcd3" />

Overall implementation looks fine, but I did notice this issue in the My Media section of the website in my local environment.

The associated component that produces this off-putting result can be found in `items/includes.js` within the components folder:

```Javascript
export function MediaItemEditLink(props){
	return ( void 0 === props.link ? null : <a href={ props.link } title="Edit media" className="item-edit-link">EDIT MEDIA</a> );
}
```

Kingcatz committed:
feat: implement bottom only rounded corners for /user/admin media thumbnails

- Apply border-radius: 0 0 6px 6px to .item-thumb and .item.video-item containers
- Add overflow: hidden to clip thumbnail images to container shape
- Use high CSS specificity to override existing styles
